doc/whats_new/v0.22.rst

```diff
@@ -460,6 +460,13 @@ Changelog
   with a target matrix `Y` in which the first column was constant.
   :issue:`13609` by :user:`Camila Williamson <camilaagw>`.
 
+:mod:`sklearn.metrics`
+..................................
+
+- |API| scoring="neg_brier_score_loss" should be used instead of
```

Comment:
I mean you should add backticks (maybe double backticks? not sure) around scoring="neg_brier_score_loss" (before "s" and after """)